guillotina/api/service.py

```diff
@@ -36,6 +42,53 @@ def __setattr__(self, name, value):
 
 
 class Service(View):
+    __validator__ = None
+    _sentinal = object()
+
+    def _get_validator(self):
```

Comment:
Change to being a `@classmethod`.

We want to cache this on the class itself, not the instance of it.

See https://www.programiz.com/python-programming/methods/built-in/classmethod for details on usage.

I think you had something close to this before and I forgot to redirect you to using a `@classmethod`. I think before you were doing something like `self.__class__.__schema__` -- what you really want to do is use classmethod.